### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by JulesKhong**
# Pull Request

Closes Issue #12 

## Issue Description
See notes in #12 

## Summary of Changes
The existing version of ts_motif had a lot of base code that needed to be deleted with new builds, tokens that needed to be renamed and the style guide was a very long HTML doc. Base code in some of the css files has been simplified with Tailwind or removed, MOTIF graphics have been removed from css files, and the style guide has been refactored so it's a series of included templates that can be edited individually, rearranged and sorted to customize the style guide and client needs. 

The style guide has also been updated to include tiles of all color swatches. 

## Testing Instructions
Expect to pull in ts_motif on a new build, and have a starter style guide with minor improvements, and more intuitive token names for line height and z-index.

### CODE REVIEW

File: assets/css/base/_generated-variables.css [hunk 1]
```diff
@@ -35,27 +31,18 @@
...
+  --lh-12: 1.2;
+  --lh-13: 1.3;
+  --lh-14: 1.4;
+  --lh-15: 1.5;
+  --lh-16: 1.6;
+  --lh-20: 2;
+  --z-0: 0;
+  --z-1: 1;
+  --z-2: 2;
+  --z-3: 3;
```
Comment (JulesKhong) [inline assets/css/base/_generated-variables.css:47]: z-0 is already in tailwind, so that can be removed.

Comment (JulesKhong) [reply assets/css/base/_generated-variables.css:47]: Maybe we need to talk in FE hour about z-index best practices?

File: assets/css/base/utilities.css [hunk 2]
```diff
@@ -1,63 +1,49 @@
...
-/* element to stretch the full width of the site.*/
-.site-container {
-  margin: 0 auto;
-  max-width: 1260px;
+/* sm:3 col, md: 4 col */
+.ts-grid-col-4 {
+  @apply sm:grid-cols-3 md:grid-cols-4 sm:grid sm:gap-md;
 }
 
 /* Mobile Padding .*/
```
Comment (JulesKhong) [inline assets/css/base/utilities.css:45]: Add to comment: Replaces .content-side-padding.

File: assets/css/common/design-elements.css [hunk 3]
```diff
@@ -1,32 +1,23 @@
...
+/*!* Wrap with .relative, eg: Hero photo style *!*/
+/*.bg-overlay-black {*/
+/*  position: absolute;*/
+/*  height: 100%;*/
+/*  width: 100%;*/
+/*  top: 0;*/
+/*  left: 0;*/
+/*  background-color: var(--color-black);*/
+/*  opacity: var(--opacity-05);*/
+/*  z-index: var(--z-index-2);*/
```
Comment (JulesKhong) [inline assets/css/common/design-elements.css:22]: Delete these design elements.

File: assets/css/common/typography.css [hunk 4]
```diff
@@ -1,151 +1,18 @@
...
-    @apply hover:underline focus:underline;
-  }
-}
-
-.text-label {
-  @apply font-semibold;
-  color: var(--color-black);
-  font-size: var(--font-size-sm);
+  font-size: 1.5625rem; /* 25px */
+  line-height: var(--lh-10);
```
Comment (JulesKhong) [inline assets/css/common/typography.css:13]: missing closing }

**Comment by jamacon36** [comment]
@JulesKhong might be good to update Tailwind to 3.0 as well but can make that it's own PR if you're onboard?

File: assets/css/base/_generated-variables.css (hunk 1 above)
Comment (JulesKhong) [reply assets/css/base/_generated-variables.css:47]: Bring font-sizes back

**Comment by mariacha** [comment]
I'm going rogue and merging this because it's 6 months old and having it sitting here is helping no one.

**Pull request merged**